transactor/src/main/scala/io/mediachain/copycat/TransactorService.scala

```diff
@@ -3,21 +3,30 @@ package io.mediachain.copycat
 import java.util.concurrent.ExecutorService
 
 import cats.data.Xor
+import io.grpc.stub.StreamObserver
 import io.grpc.{ServerBuilder, Status, StatusRuntimeException}
+import io.mediachain.copycat.Client.{ClientState, ClientStateListener}
 import io.mediachain.multihash.MultiHash
 import io.mediachain.protocol.CborSerialization
 import io.mediachain.protocol.Datastore._
-import io.mediachain.protocol.Transactor.JournalError
-import io.mediachain.protocol.transactor.Transactor.{InsertRequest, TransactorServiceGrpc, UpdateRequest}
+import io.mediachain.protocol.Transactor.{JournalError, JournalListener}
+import io.mediachain.protocol.transactor.Transactor.{MultihashReference => _, _}
```

Comment:
are we trying to avoid shadowing another MultihashReference? also yusef you keep discovering new and strange ways of aliasing imports, good job